spec/turn_spec.rb

```diff
@@ -0,0 +1,85 @@
+require './lib/turn'
+require './lib/card'
+require 'rspec'
+
+RSpec.describe Turn do
+  before :each do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+  end
+
+  it 'exists' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    expect(turn).to be_instance_of(Turn)
+  end
+
+  it 'returns a card' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    expect(turn.card).to eq(card)
+  end
+
+  it 'has a guess' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    expect(turn.guess).to eq("Juneau")
+  end
+
+  it 'correct? returns true if guess is correct' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    expect(turn.correct?).to eq(true)
+  end
+
+  it 'provides feedback when answer is correct' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    expect(turn.feedback).to eq("Correct!")
+  end
+
+  it 'incorrect? returns false if guess is incorrect' do
+    card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    turn = Turn.new("Juneau", card)
+
+    expect(turn.correct?).to eq(true)
```

Comment:
This is testing the same as line 36. Based on the title I think you might have been meaning to do something like `expect(turn.correct?).to eq(false)` which is a great idea. We want to test all possible outcomes for our code.